pulp_rpm/app/tasks/publishing.py

```diff
@@ -421,7 +421,9 @@ def publish(
 
             log.info(_("Publication: {publication} created").format(publication=publication.pk))
 
-            return publication
+            return "<Repository: {}; Version: {}>".format(
+                publication.repository.name, publication.repository_version.number
+            )
```

Comment:
While this certainly renders the result JSON serializable, the idea is to return the publication from it's serializer.

Reply:
Do we really want to serialize an entire object? How useful is that to the end user.

(NB: if that's what *is* needed, it would have been nice to have had that intent described...anywhere)

Reply:
Yes, indeed that's what we wanted in order to save a complete round trip of "oh my task is finished", "where are the created_resources", "which one of these hrefs actually looks like the thing i desired", "Oh pulp how does it really look?". Now the task can just return the think it just created.